packages/toolchain/src/server.ts

```diff
@@ -1,4 +1,4 @@
-import * as ip from 'ip'
+import * as ip from 'neoip'
```

Comment:
Any reason why `neoip` was preferred over `ip-address`?

Reply:
It is no longer maintained and has a vulnerability warning with npm. Anyways I prefer if we could implement our own ip fetching function instead of of relying on a package for something that will only be used once.